core/core-configuration-layer.el

```diff
@@ -689,7 +692,10 @@ path."
               (configuration-layer//get-latest-package-version-string
                pkg-name)))
       ;; (message "%s: %s > %s ?" pkg-name cur-version new-version)
-      (version< cur-version new-version))))
+      (if new-version
+          (version< cur-version new-version)
+        (cl-pushnew pkg-name configuration-layer--skipped-packages :test #'eq)
```

Comment:
I don't really know which one is better. I looked at the implementation for `add-to-list` and `cl-pushnew`, but it's kind of complicated so I couldn't figure out which is better. I'd expect we can trust the dosctring, no?
Besides, I saw that `cl-pushnew` is already used in `configuration-layer/create-layer`, so it's probably ok to use it.